proposals/cri-o.adoc

```diff
@@ -0,0 +1,220 @@
+== CRI-O
+
+*Name of project*: cri-o
+
+*Description*:
+
+CRI-O is a stable container runtime specifically tailored for Kubernetes.
+A founding principal was not "reinventing the wheel" and so the project uses shared components, refined approaches tested in production, and existing battle-tested code.
+Further, it is tuned for performance, stability, compatibility, and adherence to standards, particularly the Kubernetes Conformance tests.
+In short CRI-O is a building block of a Kubernetes cluster, to facilitate the life cycle of containers as required by the Kubernetes Container-Runtime-Interface (CRI).
+
+CRI-O was born in kubernetes-incubator and grew to production stable there.
+Some have claimed since being a kubernetes project that it is already "a CNCF project", but it is often left as an after thought in brand, awareness and support as a top-level project.
+This proposal is effective to promote from the kubernetes incubator a project that has fully matured.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle <jon@nstack.com>
+
+*Unique Identifier*: cri-o
+
+*License*: ALv2
+
+*Maturity Level*: Incubating
+
+*Source control repositories*: https://github.com/kubernetes-sigs/cri-o
+
+*Initial Committers*:
+
+- mrunalp
+- runcom
+- cyphar
+- mikebrow
+- sameo
+- rhatdan
+- nalind
+- giuseppe
```

Comment:
For those wondering, initial committers is made up of people from Red Hat, Suse, Intel, and IBM.

Instead of `Initial Committers` should these be labeled `Maintainers`?

Reply:
that's a fine suggestion. I was mimicking prior proposals, but they would be initial maintainers.

Reply:
Does CNCF have distinct definitions for `Committers` and `Maintainers`?